src/index.js

```diff
@@ -1,21 +1,27 @@
 const { useEffect } = require('react');
+const invariant = require('invariant');
 const { onKeyPress, convertToAsciiEquivalent, getAsciiCode } = require('./keys.js');
 
-const useKey = (callback, { detectKeys = [] } = {}, { dependencies = [] } = {}) => {
-  let allowedKeys = detectKeys;
-  if (!window || !window.document || !callback) {
-    throw new Error();
-  }
+const VALID_KEYEVENTS = ['keydown', 'keyup', 'keypress'];
```

Comment:
Please rename to `VALID_KEY_EVENTS` 
That missing `_`

Reply:
Ok!